CHANGELOG.md

```diff
@@ -111,6 +111,7 @@
 - [Named arguments syntax is now recognized in IDE][5774]. Connections to
   function arguments will now use named argument syntax instead of inserting
   wildcards on all preceding arguments.
+- [Added more user-friendly labels to dropdown widgets.][5705]
```

Comment:
Please make it more wordy. Explanations in changeling should be understandable by no-code users. "more user-friendly" does not tell much .